src/tools/examples/avtexamples/qtvtkExample.C

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+// Project developers.  See the top-level LICENSE file for dates and other
+// details.  No copyright assignment is required to contribute to VisIt.
+
+#include <GUIWindow2.h>
+
+#include <QApplication>
+
+#include <QVTKOpenGLWidget.h>
+
+#include <iostream>
+
+int
+MCVMain(int argc, char *argv[])
+{
+    int retval = 0;
+
+    //
+    // Setting the default QSurfaceFormat required with QVTKOpenGLwidget.
+    // This causes Qt to create an OpenGL 3.2 context.
+    //
+    auto surfaceFormat = QVTKOpenGLWidget::defaultFormat();
+    surfaceFormat.setSamples(0);
+    surfaceFormat.setAlphaBufferSize(0);
+    QSurfaceFormat::setDefaultFormat(surfaceFormat);
+
+    //
+    // Create the QApplication. This sets the qApp pointer.
+    //
+    QApplication *mainApp = new QApplication(argc, argv, true);
+
+    GUIWindow2 *guiWin = new GUIWindow2();
+
+    guiWin->show();
+    guiWin->raise();
+
+    retval = mainApp->exec();
+
+    return retval;
+}
+
+int
+main(int argc, char **argv)
+{
+    return MCVMain(argc, argv);
```

Comment:
Question: why not have all the logic in main? Are you planning on having other 'main' functions based on platform or some other criteria?

Reply:
I was just copying coding from some other coding I found in VisIt. It's pointless in this case. I will remove it.

Reply:
@biagas I made the changes.